core/hw/pvr/Renderer_if.h

```diff
@@ -62,6 +62,7 @@ struct Renderer
 	virtual bool Render() = 0;
 	virtual void RenderFramebuffer(const FramebufferInfo& info) = 0;
 	virtual bool RenderLastFrame() { return false; }
+	virtual void RenderVideoRouting() {}
```

Comment:
This is an internal method of renderers so I don't think it should be defined in this generic interface. I'd rather have empty, non-virtual implementations in each renderer that needs it:
```c++
void OpenGLRenderer::RenderVideoRouting()
{
#ifdef VIDEO_ROUTING
    // implementation...
#endif
}
```